app/main.py

```diff
@@ -15,5 +15,17 @@
     "Sergio": 3,
 }
 collection_of_coins = {1, 2, 25}
+all_variables = [lucky_number, pi, one_is_a_prime_number, name,
+                 my_favourite_films, profile_info, marks, collection_of_coins]
+sorted_variables = {
```

Comment:
in this task you just need to fill in the dictionary, without additional actions